ChangeLog.md

```diff
@@ -4,101 +4,46 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
-### CITA-Framework
-
-- [Optimization] Upgrade default rust toolchain to stable. [@yangby]
-- [Optimization] Remove useless dependencies. [@yangby]
-- [Optimization] Compact block Relay. [@u2] [@yangby]
+### Framework
 
 ### Executor
 
-- [Feature] Automatic execution. [@kaikai]
-- [Optimization] Enable changing size of global cache in StateDB. [@lhf]
-- [Refactor] Decouple executor and postman [@keroro520] [@WPF]
-- [Configuration] Deprecate `--genesis` command option, instead place into `executor.toml`. [@keroro520]
-- [Configuration] Add argument about timestamp uint in `executor.toml` to compatibility with Ethereum.[@zhiwei]
-- [Optimization] Change state db type to ensure safe reference. [@WPF]
-- [Optimization] Remove unused code in state db. [@WPF]
-- [Optimization] Add more tests in executor and postman. [@WPF]
-- [Optimization] Add block priority in postman. [@keroro520]
-- [Refactor] Decouple global sysconfig from transactionOptions. [@kaikai]
-- [Optimization] Deprecate some dangerous clone usage in block and state. [@keroro520]
-- [Optimization] Remove cached latest hashes. [@zhiwei]
-- [Fix] Fix problem in zk privacy. [@zhiwei]
-- [Fix] Fix defects in snapshot. [@keroro520]
-
 ### Chain
 
-- [Optimization] Rename crypto enum. [@zhiwei]
-
 ### Auth
 
-- [Optimization] Introduce quick check for history heights. [@zhiwei]
-
 ### Network
 
-- [Feature] Enable parsing hostname directly in network.toml. [@driftluo]
-- [Fix] Fix bug for network not send all msg. [@jerry-yu]
-
 ### Consensus
 
-- [Optimization] Add a min-heap timer. [@KaoImin]
-- [Optimization] Optimize wait time for proposal, prevote and precommit. [@jerry-yu]
+- [Optimization] Use crossbeam channel. [@kaikai]
```

Comment:
May "Replace std::channel with crossbeam channel." be better?

Reply:
> May "Replace std::channel with crossbeam channel." be better?

yeah.

Reply:
> May "Replace std::channel with crossbeam channel." be better?

done. Actuall it's `std::sync::mpsc::channel;`